app/controllers/model-types.js

```diff
@@ -1,14 +1,30 @@
-import Ember from "ember";
+import Ember from 'ember';
+import LocalStorageService from 'ember-inspector/services/storage/local';
 const { Controller, computed, get, inject } = Ember;
 const { sort } = computed;
-const { controller } = inject;
+const { controller, service } = inject;
 
 export default Controller.extend({
   application: controller(),
   navWidth: 180,
   sortProperties: ['name'],
+  storage: service(`storage/${LocalStorageService.SUPPORTED ? 'local' : 'memory'}`),
   options: {
-    hideEmptyModelTypes: false
+    hideEmptyModelTypes: computed({
```

Comment:
This it is now a CP, you'll need to move it outside the `options` hash into its own property on the component. When you do that, make sure to update all references to it [in the controller](https://github.com/emberjs/ember-inspector/blob/825f3de9244b952f564789d9ba2241e912e51eb8/app/controllers/model-types.js#L16-L18) and [in the toolbar](https://github.com/emberjs/ember-inspector/blob/825f3de9244b952f564789d9ba2241e912e51eb8/app/templates/model-types-toolbar.hbs#L3).